apps/freelance/src/pages/recoverpass-first-page/recoverpass-first-page.styled.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+const StyledPage = styled.div`
+  margin: auto;
+  display: flex;
+  height: 100vh;
+
+  background: url('/assets/images/top_elipse.png') no-repeat top left,
```

Comment:
if you use same image few times, better move it name to constant file and reuse instead of hardcore

Reply:
So, should I change for all the pages that are using this same image as well? Or just for this page? Sorry, I got confused how should I approach this.

Reply:
Fixed. Background Images as constant inside `global.styled.tsx`.